docs/content/config/best-practices/dkim_dmarc_spf.md

```diff
@@ -302,9 +302,7 @@ The DMARC status may not be displayed instantly due to delays in DNS (caches). D
 
     [Source][wikipedia-spf]
 
-!!! note "Disabling `policyd-spf`?"
-
-    As of now, `policyd-spf` cannot be disabled. This is WIP.
+You may disable extra and excessive SPF checks when using Rspamd via ENABLE_POLICYD_SPF=0.
```

Comment:
- Keep admonition?
- Revised phrasing and used `inline block` styling for the ENV.
- Linked to referenced ENV in our docs.

```suggestion
!!! tip "Disabling the default SPF service `policy-spf`"

    Set [`ENABLE_POLICYD_SPF=0`][docs-env-spf-policyd] to opt-out of the default SPF service. Advised when Rspamd is configured to handle SPF instead.
    
[docs-env-spf-policyd]: ../environment.md#enable_policyd_spf
```